third-party/crowdsec/sample-configuration/parsers/james-auth.yaml

```diff
@@ -73,3 +74,19 @@ nodes:
           expression: evt.Parsed.mdc_remoteIP
         - meta: user
           expression: evt.Parsed.mdc_username
+  - grok:
+      name: "DICTIONARY_ATTACK"
```

Comment:
Maybe rename the file james.yml? It no longer is just about auth...

Reply:
@chibenwa  A file yaml is a parser, so how about creating a new parser for Dictionary attack?

Reply:
Ok